### PR TITLE
Readme Update for Quickwit 0.4

### PR DESCRIPTION
**Pull request by maximkpa**
Description: 

- Wording update (more condensed bullet points, changed order)
- Minor fixes (ex. emoji in the Search more with less to create structure, etc.)
- Third-party integrations image instead of icons

### CODE REVIEW

**Comment by ChillFish8** [comment]
image

I think there are some funky layouts going on in the markdown.

**Comment by maximkpa** [comment]
> image
> 
> I think there are some funky layouts going on in the markdown.



> image
> 
> I think there are some funky layouts going on in the markdown.

Thank you @ChillFish8 for noticing, just fixed it!

File: README.md [hunk 1]
```diff
@@ -39,33 +41,29 @@ Quickwit is a cloud-native search engine for log management & analytics. It is d
...
-  - Distributed and replicated ingestion queue
-  - Tiered storage (local drive, block storage, object storage)
-  - Native support for OpenTelemetry exporters (logs and traces)
-  - Jaeger integration
+- [Quickwit 0.5 - Q1 2023](https://github.com/quickwit-oss/quickwit/projects/6)
   - Grafana data source
+  - Native support for OpenTelemetry
+  - REST API for managing indexes
 - [Long-term roadmap](ROADMAP.md)
+  - Distributed indexing
```
Comment (ChillFish8) [inline README.md:64]: Maybe this needs clarifying as just above we mention we support distributed indexing with kafka.

**Review by ChillFish8 — APPROVED** [review]
Just the one comment about clarity.

File: README.md [hunk 2]
```diff
@@ -39,33 +41,29 @@ Quickwit is a cloud-native search engine for log management & analytics. It is d
...
-- Kafka-native ingestion
-- Search stream API that notably unlocks full-text search in ClickHouse
+- Distributed search
+- Distributed indexing with Kafka
+- Native support for Kafka & Kinesis with exactly-once semantics
+- Schemaless or strict schema indexing
+- Works out of the box with sensible defaults
+- Lightweight Embedded UI
 
 ### 🔮 Roadmap
```
Comment (guilload) [inline README.md:58]: Please, leave the current roadmap as is:

[Quickwit 0.5 - Q2 2023](https://github.com/quickwit-oss/quickwit/projects/6)
- Distributed and replicated ingestion queue
- Tiered storage (local drive, block storage, object storage)
- Native support for OpenTelemetry exporters (logs and traces)
- Jaeger integration
- Grafana data source

[Long-term roadmap](https://github.com/quickwit-oss/quickwit/blob/main/ROADMAP.md)
- Pipe-based query language
- Security (TLS, authentication, RBAC)
- Transforms
[and more...](https://github.com/quickwit-oss/quickwit/blob/main/ROADMAP.md)

File: README.md (hunk 1 above)
Comment (maximkpa) [reply README.md]: All good now, the new roadmap doesn't have this part

**Review by fmassot — APPROVED** [review]

**Pull request merged**